x-pack/test/apm_api_integration/tests/throughput/service_maps.spec.ts

```diff
@@ -46,10 +45,8 @@ export default function ApiTest({ getService }: FtrProviderContext) {
 
     const serviceInventoryThroughput = serviceInventoryAPIResponse.body.items[0].throughput;
 
-    const serviceMapsNodeDetailsThroughput = meanBy(
-      serviceMapsNodeDetails.body.currentPeriod.transactionStats?.throughput?.timeseries,
-      'y'
-    );
+    const serviceMapsNodeDetailsThroughput =
```

Comment:
This is good, but do you mind also adding an explicit test rather than just comparing it to the output of other API endpoints?